src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, Drawable {
+    private String color;
+    private double width;
+    private double height;
```

Comment:
```suggestion
```

Reply:
I do not agree, because I initialize method getArea and I need width and height in order to have default method for figure